1-html/README.es.md

```diff
@@ -5,7 +5,7 @@
 
 - Cree el repositorio `goit-markup-hw-01`.
 - Realice un marcado HTML para el diseño de página
-  [**la tarea de casa #1**](<https://www.figma.com/file/oTYBECAN79dXy19hzWObO4/Web-Studio-(Version-2.1)?node-id=0%3A1>),
+  [**la tarea de casa #1**](<https://www.figma.com/file/oHmmiFKINiQlrPDLpPPSse/Web-Studio-ESP?node-id=0%3A1>),
```

Comment:
Design should be in english cause it's being updated quite often. We can't support multiple copies.